tests/test_types.py

```diff
@@ -1806,9 +1806,18 @@ class MyModel(BaseModel):
     'kwargs,type_,a',
     [
         ({'pattern': '^foo$'}, int, 1),
-        ({'gt': 0}, conlist(int, min_length=4), [1, 2, 3, 4, 5]),
-        ({'gt': 0}, conset(int, min_length=4), {1, 2, 3, 4, 5}),
-        ({'gt': 0}, confrozenset(int, min_length=4), frozenset({1, 2, 3, 4, 5})),
+        *[
+            ({constraint_name: 0}, conlist(int, min_length=4), [1, 2, 3, 4, 5])
```

Comment:
What's the value in having the `conlist` and `min_length` here? Can we just do `list[int]`, `set[int]`, etc?

Reply:
Yeah, let me use `list[int], set[int]` instead 👍